INTHEWILD.md

```diff
@@ -0,0 +1,113 @@
+## Superset users in the wild
+
+Here's a list of organizations, broken down into broad industry categories, that have taken the time to send a PR to let
+the world know they are using Apache Superset. If you are a user and want to be recognized,
+all you have to do is file a simple PR [like this one](https://github.com/apache/incubator-superset/pull/7576). If you think
```

Comment:
This pr is probably out of date now, maybe we should remove it?

Reply:
Or replace it with another PR that adds a company to this file, once this has been merged in

Reply:
Updated the PR example to a recent one

Reply:
We probably should just remove this link altogether, then add one back in once someone has edited this file post merge